boards/px4/fmu-v4/init/rc.board_defaults

```diff
@@ -9,6 +9,14 @@ then
 	# Disable safety switch by default
 	param set CBRK_IO_SAFETY 22027
 
+	# start MAVLink on Wifi (ESP8266 port). Except for the TealOne airframe.
+	if ! param compare SYS_AUTOSTART 4250
+	then
+		param set MAV_2_CONFIG 301
```

Comment:
Would be good if we had transitional support, such that the parameters are automatically set for existing setups, not only on AUTOCNF. E.g. by checking for `MAV_2_CONFIG == 0`.

Reply:
If I use DSHOT_TEL_CFG and want to disable the `MAV_2` mavlink instance, I'd set `MAV_2_CONFIG = 0` and `DSHOT_TEL_CFG = 301`. So if I check for `MAV_2_CONFIG == 0`, I would create a conflict at every boot, no? Or do you mean if AUTOCONF and `MAV_2_CONFIG == 0`?

Reply:
Right, that does not work. We could set `SER_WIFI_BAUD` to a default that is not used otherwise (e.g. 1) and check for that.

Reply:
Is this what you meant? (if so, I'll squash and rebase)

Reply:
Exactly. I'd keep the part in `AUTOCNF` as well, so we can just remove the transitional logic after the next release.